CHANGELOG.md

```diff
@@ -4,7 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 The following changes have been implemented but not released yet:
 
-## [Unreleased]
+## [0.2.0] - 2021-12-15
```

Comment:
I would keep the `Unreleased` header so that we add new unreleased changelog entries under it, but it's not a big deal if it is just added back with the next changes.